doc/nrf/releases_and_maturity/releases/release-notes-2.5.1.rst

```diff
@@ -0,0 +1,96 @@
+To learn more, see :ref:`software_maturity`.
+
+Highlights
+**********
+
+This patch release adds changes on top of the :ref:`nRF Connect SDK v2.5.0 <ncs_release_notes_250>`.
+The changes effect Matter, `nrfx`_ v3.2.0, Wi-Fi, TF-M, :ref:`liblwm2m_carrier_readme` library, and modem firmware v1.3.6.
+
+See :ref:`ncs_release_notes_251_changelog` for the complete list of changes.
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v2.5.1**.
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` and :ref:`gs_updating_repos_examples` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions for v2.5.1`_.
+
+IDE and tool support
+********************
+
+`nRF Connect for Visual Studio Code extension <nRF Connect for Visual Studio Code_>`_ is the only officially supported IDE for |NCS| v2.5.1.
+SEGGER Embedded Studio Nordic Edition is no longer tested or recommended for new projects.
+
+:ref:`gs_app_tcm`, used to install the |NCS| automatically from `nRF Connect for Desktop`_, is available for Windows, Linux, and macOS.
+
+Supported modem firmware
+************************
+
+See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.5.1_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.5.1_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.5.1`_ for the list of issues valid for the latest release.
+
+.. _ncs_release_notes_251_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+nrfx
+====
+
+* Updated the nrfx QSPI driver behavior.
+  Now QSPI peripheral is not activated during the driver initialization but rather when the first operation is requested or when the :c:func:`nrfx_qspi_activate` function is called.
+* Fixed an issue when external memory requires a relatively long time for the operation to happen.
+  This is done by adding a timeout member to :c:struct:`nrfx_qspi_config_t` , which can be configured at runtime timeout.
+  If the timeout value is ``0``, the old value is used instead (500 milliseconds).
+
+Zephyr
+======
```

Comment:
An entry that mentions the workaround for anomaly 168 should be added somewhere here. Perhaps in a similar form as was used in [v2.1.4](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/2.4.2/nrf/releases/release-notes-2.1.4.html#zephyr).